### ISSUE TITLE
How to view 'No records available' as a row if data object is empty ?

### ISSUE THREAD

**Issue by tony-stark001** [joaocarmo/react-smart-data-table#13]
Currently the column headers are disappearing if data object is empty. I want column headers fixed, if empty data object provided. Also how to view 'No records found' text if data object is empty. ?

**Comment by joaocarmo** [comment]
The newly released `v0.6.5` should address the _second issue_. You can now pass any renderable object to the prop `emptyTable` which will be presented whenever there is no data. Please, try it in the [live demo](https://joaocarmo.github.io/react-smart-data-table/examples/semantic-ui/).

Regarding the _first issue_, I'll keep this open and think about it. Though I'm inclined to not implement such a thing, because if the table is supposed to be smart and infer the column names from the data and there is no data, then there can be no headers (there's no information to be used to create them). The solution would be to supply your own information of _default headers_ and for that, there are many better tables out there. 

**TL;DR**
The _first issue_ goes against the essential philosophy of a smart data table.
